lib/compat/wordpress-6.1/class-wp-theme-json-6-1.php

```diff
@@ -566,9 +577,19 @@ private static function get_block_nodes( $theme_json, $selectors = array() ) {
 					if ( array_key_exists( $element, static::VALID_ELEMENT_PSEUDO_SELECTORS ) ) {
 						foreach ( static::VALID_ELEMENT_PSEUDO_SELECTORS[ $element ] as $pseudo_selector ) {
 							if ( isset( $theme_json['styles']['blocks'][ $name ]['elements'][ $element ][ $pseudo_selector ] ) ) {
+
+								$block_selector = [];
+								// This converts selectors like '.wp-element-button, .wp-block-button__link'
```

Comment:
Since we do the same thing twice, should we make it a function?

Reply:
I don't know! I've seen a lot of similar repetition over the class, and seen no similar utility functions around. It makes sense to me, though. Should it be private or protected?

Reply:
private unless its needed outside the class.

Reply:
have a look when you can!